scalafmt-sysops/jvm/src/main/scala-2.12/org/scalafmt/CompatCollections.scala

```diff
@@ -0,0 +1,10 @@
+package org.scalafmt
+
+private[scalafmt] object CompatCollections {
+  val JavaConverters = scala.collection.JavaConverters
+  object ParConverters {
+    implicit class XIterable[T](val col: Iterable[T]) extends AnyVal {
```

Comment:
i think in scalameta code there's a convention of what to call these implicit extension classes. i forgot but you can easily find existing examples. I thought it was a bit longer than just `X`, with all due respect to Shmelon Shmusk.